### ISSUE TITLE
Support image expansion for those hosted by StackExchange's imgur account

### ISSUE THREAD

**Issue by vermiculus** [honestbleeps/Reddit-Enhancement-Suite#2000]
See [this comment on Reddit](http://www.reddit.com/r/LaTeX/comments/2u7yg8/someone_just_told_me_about_latex_i_dont_get_it/co6rseq)

Source:
```markdown
See this
[visual comparison](http://i.stack.imgur.com/OBiLO.png) and the full-blown
[analysis](http://i.stack.imgur.com/3ScJ0.png) from
[TeX.SX](http://tex.stackexchange.com/a/110140/17423).
```

Notice how they are all on the `i.stack.imgur.com` domain. I believe we can add this (or something else, perhaps just `stack.imgur.com`) to `domains` by using [`showImages.js:2098`](https://github.com/honestbleeps/Reddit-Enhancement-Suite/blame/fe00501ec91460ea8a07a0a157c5dd843ab2fcaf/lib/modules/showImages.js#L2098) (introduced in commit 4e10bd20355c7926141a026bd82fe31a8257372f), but I am unfamiliar with your architecture.

**Comment by andytuba** [comment]
Thanks for highlighting this. The architecture is a little byzantine :grin:  We should do a more generic solution that supports i.anyone.imgur.com.

**Comment by vermiculus** [comment]
Any further thoughts on this?

**Comment by allthefoxes** [comment]
Should be straightforward to implement. You bored @erikdesjardins ?

**Comment by andytuba** [comment]
FYI, my understanding is that imgur won't add any new `i.foo.imgur.com` customers.

**Comment by vermiculus** [comment]
Any further thoughts on this?

**Comment by andytuba** [comment]
thanks for the bump!  i'll take care of this .. sometime this week.

**Comment by andytuba** [comment]
fixed for next release

**Comment by vermiculus** [comment]
:+1: thanks!

**Comment by erikdesjardins** [comment]
Fixed by #2239, should be closed.

**Comment by vermiculus** [comment]
Looks good to me.
